interfaces/connection.go

```diff
@@ -49,29 +50,51 @@ type ConnectedSlot struct {
 // Attrer is an interface with Attr getter method common
 // to ConnectedSlot, ConnectedPlug, PlugInfo and SlotInfo types.
 type Attrer interface {
-	Attr(key string, val interface{}) error
+	// Attr returns attribute value for given path, or an error. Dotted paths are supported.
+	Attr(path string, val interface{}) error
+	// Lookup returns attribute value for given path, or false. Dotted paths are supported.
```

Comment:
not sure this is super clear,  is false the value of the interface or of the bool?  maybe naming the return argument would also help  (value interface{}, ok bool)